assets/js/ga.js

```diff
@@ -0,0 +1,14 @@
+var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
```

Comment:
This is over-engineering the check: the idea is that we want the next `if` block to kick in if DNT is not enabled, so based on [the MDN docs for DNT](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack) we can achieve that with with a direct check:
```
const doNotTrack = navigator.doNotTrack || navigator.msDoNotTrack || window.doNotTrack;
if (!doNotTrack || doNotTrack === 'no') {
  ...
}
```
(note `window` rather than `navigator` is used by IE11, Edge, and Safari 7.1.3+... yay standards!)